site/templates/collection/carousel_item.jet

```diff
@@ -13,7 +13,7 @@
     <a href="{{routeToSlug(item.Slug)}}" class="carousel-item-link">
 
       {{if isset(item.Images["Carousel"])}}
-        <s72-image src="{{item.Images.Carousel}}" alt="{{item.Title}}" class="carousel-item-image"></s72-image>
+        <div class="carousel-item-image" style="background-image: url({{item.Images.Carousel}})"></div>
```

Comment:
Ideally, I would have used `object-fit: cover;` on the image but it isn't supported in IE11. The tradeoff with this method is that you don't get the alt tag. Alternatively, I discovered that you can use `background-image` on a `<img>` tag but I'm not sure if that is good practice?